RobotCode/src/main/java/IntakeCommand.java

```diff
@@ -0,0 +1,32 @@
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+public class IntakeCommand extends Command
+{
+    private IntakeSubsystem intake;
+    private JoystickButton toggleIntakeButton;
```

Comment:
This line and the next can be compressed into `private JoystickButton toggleIntakeButton, spinIntakeButton`.